source

```diff
@@ -10558,6 +10558,20 @@ partial interface <dfn id="document" data-lt="">Document</dfn> {
   <p>Each <code>Document</code> has a <dfn data-x="concept-document-navigation-id">navigation
   id</dfn>, which is a <span>navigation ID</span> or null, initially null.</p>
 
+  <p>Each <code>Document</code> has a <dfn
+  data-x="concept-document-bfcache-blocking-reasons">bfcache blocking reasons</dfn>, which is a
```

Comment:
I think we should be consistent with whether we use the term "bfcache" or "back/forward cache". It seems like https://html.spec.whatwg.org/multipage/browsing-the-web.html#reactivate-a-document uses the former, so maybe we should stick to that? Also maybe link to the bfcache note.